source/user-manual/api/getting-started.rst

```diff
@@ -63,11 +76,168 @@ Wazuh API endpoints require authentication in order to be used. Therefore, all c
             "timestamp": "2020-05-25T07:05:00+0000"
         }
 
-Once we are logged in we can run any endpoint following the structure below. Please, do not forget to replace <endpoint> and <YOUR_JWT_TOKEN> with your own values.
+
+Once we are logged in we can run any endpoint following the structure below. Please, do not forget to replace <endpoint> with your own value. In case you are not using the environment variable, replace $TOKEN with your jwt token.
```

Comment:
```suggestion
Once logged in, it is possible to run any endpoint following the structure below. Please, do not forget to replace <endpoint> with your own value. In case you are not using the environment variable, replace $TOKEN with your jwt token.
```